packages/hdwallet-core/src/ethereum.ts

```diff
@@ -107,6 +108,18 @@ export interface ETHVerifyMessage {
   signature: string;
 }
 
+export type ETHSignTypedData = {
+  hashableData: TypedData;
+  addressNList: BIP32Path;
+};
+
+export type ETHSignedTypedData = {
```

Comment:
@pastaghost is there a reason for returning the `domainSeparatorHash` and `messageHash` as opposed to just the address + sig like in `signMessage`?